cypress/integration/crud.spec.js

```diff
@@ -1,36 +1,21 @@
+import locators from "../commons/locators";
+
 describe('Notes', () => {
   beforeEach(() => {
-    cy.visit('http://notes-serverless-app.com')
-
-    cy.get('.navbar-nav a:contains(Login)').click()
-
-    cy.get('#email').type(Cypress.env('user'))
-    cy.get('#password').type(Cypress.env('password'))
-    cy.get('button[type="submit"]').click()
+    cy.visit('/')    
+    cy.loginAs()
```

Comment:
Que tal chamar o comando somente de `login`?
O nome `loginAs` fica estranho quando nenhum parâmetro é passado, não acha?

Reply:
A ideia era justamente essa...caso for necessário efetuar o login com outras credenciais, ficar parecido como "logar-como"...vou alterar aqui